.github/workflows/build_and_test.yml

```diff
@@ -252,7 +252,7 @@ jobs:
     - name: Install Python packages (Python 3.8)
       if: (contains(matrix.modules, 'sql') && !contains(matrix.modules, 'sql-'))
       run: |
-        python3.8 -m pip install 'numpy>=1.20.0' 'pyarrow<7.0.0' pandas scipy xmlrunner
+        python3.8 -m pip install 'numpy>=1.20.0' pyarrow pandas scipy xmlrunner
```

Comment:
I saw in the description it said `numpy 1.22.2`. This seems only asking for 1.20.0+?

Reply:
Oh oops, I missed this part. Thanks @dongjoon-hyun for addressing this comment.